### DISCUSSION TITLE
Linux Need Realtime Backup Similar to Macrium Reflect for Windows

### DISCUSSION THREAD

**Discussion by sparks79** [RsyncProject/rsync#463] [General]
I am Using WindowsFx with a 1TB M.2 Drive.
I have a Spare 1 TB M.2 Drive already Installed.
I would like to have My Linux Distro ( WindowsFx ) Backed up Consistantly .
By that I mean a Realtime Backup of my WindowsFx Running in the Background and Backing Up Everything from the Linux Drive to the Spare Drive.
So that in the event of some System Failure, I can quickly Switch Over to the Spare Drive and Run Linux from the last known Safe Worrking Backup Instance.
Does anyone know of Suitable Software for My Requirements.

**Comment by tmknight** [comment]
I suggest you look into [disk mirroring](https://en.m.wikipedia.org/wiki/Disk_mirroring)
